content/docs/dvclive/integrations/pytorch.md

```diff
@@ -0,0 +1,67 @@
+# PyTorch
```

Comment:
I like the fact that we have fully functional example for `pytorch` usage, but I am not sure it is actually an integration. This word insinuates that there is some automagic allowing some default behavior, while we are showing here "normal" use case (using `log`, `next_step`)

Reply:
I see your point. Maybe we could replace the word `integrations` with `tutorials`? Would `tutorial` still have a proper meaning for the other examples where there is an actual "integration"?

Reply:
I also remember that @jorgeorpinel was not 100% sure about using "integrations" (https://github.com/iterative/dvclive/issues/87#issuecomment-864475219):

> Hmmm maybe I'm confused about the term "integration" here. The only other software that DVCLive integrates with is DVC.

Reply:
Tutorial sounds good for pytroch, but then it wouldn't describe exactly what we do with Keras / MMCV - thats why I said I am not sure what we could do here.

Reply:
I think it would be nice to add a PyTorch Lightning integration and then handle it the same way as TF/keras since they are sort of analogous, right?

Reply:
@dberenbaum Do you mean that we should omit the "raw" `Pytorch` example and replace it with a `Pytorch Lighting` integration?

They are sort of analogous indeed but the main difference would be that `keras` is distributed as part of TensorFlow, maintained by the TensorFLow org and mentioned as the "official" high level API while `PyTorch Lightning` is, as of today, a separated library not distributed as part of PyTorch and neither maintained by the PyTorch organization. Although it seems to be the most popular high level API for PyTorch user these days.

Reply:
Good point, that does make it a bit weird to only include a Lightning example. Both the TF and PyTorch sections could include both low-level examples and high-level keras/lightning integrations.

How much value do you think a raw PyTorch example provides? It seems obvious once you are familiar with the `dvclive.log(); dvclive.next_step()` workflow, but I could see it being easier to immediately jump to a PyTorch example without reading any of the other docs first.

Also, we could make this whole section more generic. Instead of "integrations" or "tutorials," it could be called "ML frameworks" or something like that.

Reply:
I'm not sure how many people actually write their own low level loop in TensorFlow or Pytorch these days, but agree that it could be useful, even just for SEO, to have both examples. I will add a TensorFlow example as part of this P.R. The Lightning example would need to wait for the integration to be added.

*ML Frameworks* sound good to me.